### PR TITLE
added cloudflare wrangler and script for cloudflare to run for deployments

### PR DESCRIPTION
**Pull request by dweng0**
This PR now adds a Github action to handle deployment to Cloudflare, see <https://github.com/autonity/docs.autonity.org/issues/227> for more details about this.

### CODE REVIEW

**Comment by aiman** [comment]
Probably this PR should be changed to draft to reflect the fact its work in progress, we don't need to merge it until decision is made on how to proceed

File: .github/workflows/cloudflare-preview.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+on:
+    pull_request:
+        branches:
+        - develop   
+name: Cloudflare-preview
+jobs:
+    build-deploy:
+      runs-on: ubuntu-latest
```
Comment (szemate) [inline .github/workflows/cloudflare-preview.yml]: The indentation is messed up here

**Review by szemate — COMMENTED** [review]
I like this approach, and it's great that each step could be handled by a 3rd-party action

File: .github/workflows/cloudflare-preview.yml (hunk 1 above)
Comment (dweng0) [reply .github/workflows/cloudflare-preview.yml]: Have updated

**Review by szemate — DISMISSED** [review]
Looks good!

**Comment by aiman** [comment]
Closing to help diagnose CI issues. Will reopen when ready for review.

**Pull request closed without merging**